src/lib/math.js

```diff
@@ -314,6 +314,55 @@ const $builtinmodule = function (name) {
         throw new Sk.builtin.NotImplementedError("math.isqrt() is not yet implemented in Skulpt");
     };
 
+    function lcm() {
+        const nargs = arguments.length;
+
+        // lcm() without arguments returns 1
+        if (nargs === 0) return new Sk.builtin.int_(1);
+
+        let result = arguments[0];
+        Sk.builtin.pyCheckType("*integers", "integer", Sk.builtin.checkInt(result));
+        result = Sk.builtin.asnum$(result);
+
+        if (nargs === 1) {
+            if (typeof result === "number") {
+                return new Sk.builtin.int_(Math.abs(result));
+            } else {
+                return new Sk.builtin.int_(result);
```

Comment:
I added this check twice, also at the end. Not sure it it is required though.

Reply:
Your `abs` function will need to behave different depending on whether you have a number or a JSBI.BigInt.

Another thing to note in this implementation is that - since lcm will keep growing in size - you will need to keep checking that the result hasn't grown larger than `Number.MAX_SAFE_INTEGER`. As soon as the result starts getting bigger than that you'll need to start working with JSBI.BigInts.